src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -34,6 +34,8 @@ public abstract class TableValue : ValidFormulaValue
 
         public abstract IEnumerable<DValue<RecordValue>> Rows { get; }
```

Comment:
>IEnumerable<DValue<RecordValue>> Rows  [](http://example.com/codeflow?start=24&length=38)

With the change, do we want to add IAsyncEnumerable RowsAsync?

Reply:
Maybe